.github/workflows/test_ert_with_slurm.yml

```diff
@@ -71,6 +71,36 @@ jobs:
       run: |
         set -e
 
-        cd test-data/poly_example
-        echo "QUEUE_SYSTEM SLURM" >> poly.ert
+        cp -a test-data/poly_example poly_slurm
+        pushd poly_slurm
+
+        cat << EOF >> poly.ert
+        NUM_REALIZATIONS 5
+        QUEUE_SYSTEM SLURM
+        EOF
+
+        time ert ensemble_experiment poly.ert
+
+
+    - name: Test Torque driver through slurm
+      run: |
+        set -e
+
+        sudo apt install slurm-wlm-torque
+
+        # Patch qstat from slurm-wlm-torque (patch is submitted upstream):
+        cp /usr/bin/qstat qstat
+        # shellcheck disable=SC2016
+        sed -i '516 i printf("    Exit_status = %s\\n", $job->{"exit_code"});' qstat
+        cp -a test-data/poly_example poly_torque
+
+        cat << EOF >> poly_torque/poly.ert
+        NUM_REALIZATIONS 5
+        QUEUE_SYSTEM TORQUE
+        QUEUE_OPTION TORQUE QSTAT_CMD $(pwd)/qstat
+        QUEUE_OPTION TORQUE QSTAT_OPTIONS -f
```

Comment:
I get the `QSTAT_CMD`, but is `-f ` required by SLURM?

Reply:
Yes, it means "full output" which is the only thing our parser can parse. We need to set it, because the default is "-f -x", while the "-x" option is unsupported in slurm-wlm-torque (thus we should not have it as default in the driver, but in ert-configurations)

Reply:
added comment on qstat_options and keep_qsub_output

Reply:
nice!